man/bffindfw.8

```diff
@@ -0,0 +1,12 @@
+.TH BFFINDFW 8 "July 2020"
+.SH NAME
+bffindfw \- Print the path to network adapter FW for current platform
+.SH SYNOPSIS
+.B bffindfw
```

Comment:
Just a small comment:

It's ok for now. In the future when we need to add more utility code for FW, we could consider using a more generic name (such as bfhcafw) with command line arguments to do different things.

Reply:
I'll just rename it now to reduce potential pain in the future